.circleci/config.yml

```diff
@@ -0,0 +1,96 @@
+version: 2.1
+
+parameters:
+  # This is the Go version of the CircleCI image.
+  #
+  golang-version:
+    type: string
+    default: "1.20"
+
+  # This is the golangci-lint version we use.
+  #
+  # Take a look at the official docs to check if there is a newer version
+  # available:
+  #
+  # https://golangci-lint.run/usage/install/#other-ci
+  # https://github.com/golangci/golangci-lint/releases
+  #
+  golangci-lint-version:
+    type: string
+    default: "v1.51.2"
```

Comment:
Should we update this to the latest version? v1.52.2 is now available: https://github.com/golangci/golangci-lint/releases/tag/v1.52.2

Reply:
Addressed in a068c86